src/app/host/[roomId]/page.tsx

```diff
@@ -1,32 +1,40 @@
 'use client'
 
-import Link from 'next/link'
 import { useEffect, useState } from 'react'
 import { socketEmitter } from '@/services/socket'
-import { useSocketEmitterLocal } from '@/services/socket'
+// import { useSocketEmitterLocal } from '@/services/socket'
 import { useSocketListener } from '@/services/socket'
 import type { ListenerRes } from '@/services/socket'
 import HostControlButton from '@/components/HostControlButton'
 import HostSettingsButton from '@/components/HostSettingsButton'
 import RoomMainUi from '@/components/RoomMainUi'
 import RoomInfo from '@/components/RoomInfo'
+import HostDevButtons from '@/components/socketIoDevTools/HostDevButtons'
+import { namesArray } from '@/utils/sampleData'
 
-export default function HostRoom({ params }: { params: { roomId: string } }) {
-	const [allUserPointData, setAllUserPointData] = useState<ListenerRes[]>([])
+// NOTE Story point codes:
```

Comment:
suggestion: nice you have codes. you could literally store this in a const

```js
export const POINT_CODES: {
  JOIN: -99,
  RESET: -55
 // etc..
}
```

and then use it like 	`socketEmitter('join-room', { roomId: roomId, message: POINT_CODES.JOIN, userName: hostName })`

btw, using always numbers, and negative numbers for "codes" was a very smart idea 💯 . 
why not make the "?" also a negative number?

Reply:
DUH!! Of course, store these in a const!! They are CONSTANTS after all!! Great suggestion. 

The "?" is already a negative number, -1. So the story point sequence is -1, 0, 1, 2, 3, 5, ...